submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style-type: none;
+    text-decoration: none;
```

Comment:
Also, let's add this `text-decoration` for resetting styles for `a` (not to all elements)